python/cudf/cudf/core/column/datetime.py

```diff
@@ -225,6 +225,9 @@ def get_dt_field(self, field: str) -> ColumnBase:
     def ceil(self, field: str) -> ColumnBase:
         return libcudf.datetime.ceil_datetime(self, field)
 
+    def floor(self, field: str) -> ColumnBase:
+        return libcudf.datetime.floor_datetime(self, field)
```

Comment:
Let's name this [freq](https://pandas.pydata.org/docs/reference/api/pandas.DatetimeIndex.ceil.html) for consistency.
```suggestion
    def ceil(self, freq: str) -> ColumnBase:
        return libcudf.datetime.ceil_datetime(self, freq)

    def floor(self, freq: str) -> ColumnBase:
        return libcudf.datetime.floor_datetime(self, freq)
```